This patch is the reverse of:

commit 3d62817080598d68514ed2691b1f616fcda2abe3
Author: Philip Kime <Philip@kime.org.uk>
Date:   Fri Nov 11 00:07:58 2016 +0100

    Updated U::C which needed test updates

diff --git a/Build.PL b/Build.PL
index 99cac079..3cbf404e 100644
--- a/Build.PL
+++ b/Build.PL
@@ -57,7 +57,7 @@ my $builder = $class->new(
                  'Mozilla::CA' =>  '20160104',
                  'Regexp::Common' => 0,
                  'Log::Log4perl' => 0,
-                 'Unicode::Collate' => '1.19',
+                 'Unicode::Collate' => '1.14',
                  'Unicode::Normalize' => '1.25',
                  'Unicode::LineBreak' => '2016.003',
                  'Unicode::GCString' => 0,
diff --git a/t/annotations.t b/t/annotations.t
index b5fd6499..07c8d58d 100644
--- a/t/annotations.t
+++ b/t/annotations.t
@@ -1,4 +1,3 @@
-
 # -*- cperl -*-
 use strict;
 use warnings;
@@ -71,7 +70,7 @@ my $ann1 = q|    \entry{ann1}{misc}{}
       \strng{authornamehash}{90ae96c82de92e36949bc64254bbde0c}
       \strng{authorfullhash}{90ae96c82de92e36949bc64254bbde0c}
       \field{sortinit}{L}
-      \field{sortinithash}{7bba64db83423e3c29ad597f3b682cf3}
+      \field{sortinithash}{872351f18d0f736066eda0bf18bfa4f7}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{title}{The Title}
diff --git a/t/basic-misc.t b/t/basic-misc.t
index aaaf857c..740d171f 100755
--- a/t/basic-misc.t
+++ b/t/basic-misc.t
@@ -94,7 +94,7 @@ my $u1 = q|    \entry{u1}{misc}{}
       \strng{authorfullhash}{b78abdc838d79b6576f2ed0021642766}
       \field{labelalpha}{AAA\textbf{+}00}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -211,7 +211,7 @@ my $murray1 = q|    \entry{murray}{article}{}
       \strng{authorfullhash}{61836f4684b2615842b68c26479f6ec2}
       \field{labelalpha}{Hos\textbf{+}98}
       \field{sortinit}{H}
-      \field{sortinithash}{2f664b453ec75da1fe3804ca92633405}
+      \field{sortinithash}{82012198d5dfa657b8c4a168793268a6}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -312,7 +312,7 @@ my $murray2 = q|    \entry{murray}{article}{}
       \strng{authorfullhash}{61836f4684b2615842b68c26479f6ec2}
       \field{labelalpha}{Hos98}
       \field{sortinit}{H}
-      \field{sortinithash}{2f664b453ec75da1fe3804ca92633405}
+      \field{sortinithash}{82012198d5dfa657b8c4a168793268a6}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -349,7 +349,7 @@ my $t1 = q+    \entry{t1}{misc}{}
       \strng{authorfullhash}{858fcf9483ec29b7707a7dda2dde7a6f}
       \field{labelalpha}{Bro92}
       \field{sortinit}{B}
-      \field{sortinithash}{5f6fa000f686ee5b41be67ba6ff7962d}
+      \field{sortinithash}{4ecbea03efd0532989d3836d1a048c32}
       \true{uniquework}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -375,7 +375,7 @@ my $t2 = q|    \entry{t2}{misc}{}
       \strng{authorfullhash}{858fcf9483ec29b7707a7dda2dde7a6f}
       \field{labelalpha}{Bro94}
       \field{sortinit}{B}
-      \field{sortinithash}{5f6fa000f686ee5b41be67ba6ff7962d}
+      \field{sortinithash}{4ecbea03efd0532989d3836d1a048c32}
       \true{uniquework}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -405,7 +405,7 @@ my $anon1 = q|    \entry{anon1}{unpublished}{}
       \strng{shortauthorfullhash}{9873a6cc65c553faa2b21aaad626fe4b}
       \field{labelalpha}{XAn35}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{shortauthor}
@@ -441,7 +441,7 @@ my $anon2 = q|    \entry{anon2}{unpublished}{}
       \strng{shortauthorfullhash}{f64c29e89ea49402b997956610b58ef6}
       \field{labelalpha}{YAn39}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{shortauthor}
@@ -472,7 +472,7 @@ my $url1 = q|    \entry{url1}{misc}{}
       \strng{authorfullhash}{b2106a3dda6c5a4879a0cab37e9cca55}
       \field{labelalpha}{Ali05}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \field{extraalpha}{4}
       \field{labelnamesource}{author}
       \field{year}{2005}
@@ -637,7 +637,7 @@ my $isbn1 = q|    \entry{isbn1}{misc}{}
       \strng{authorfullhash}{f6595ccb9db5f634e7bb242a3f78e5f9}
       \field{labelalpha}{Flu}
       \field{sortinit}{F}
-      \field{sortinithash}{276475738cc058478c1677046f857703}
+      \field{sortinithash}{c6a7d9913bbd7b20ea954441c0460b78}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{isbn}{978-0-8165-2066-4}
@@ -658,7 +658,7 @@ my $isbn2 = q|    \entry{isbn2}{misc}{}
       \strng{authorfullhash}{f6595ccb9db5f634e7bb242a3f78e5f9}
       \field{labelalpha}{Flu}
       \field{sortinit}{F}
-      \field{sortinithash}{276475738cc058478c1677046f857703}
+      \field{sortinithash}{c6a7d9913bbd7b20ea954441c0460b78}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
       \field{isbn}{978-0-8165-2066-4}
@@ -696,7 +696,7 @@ my $clone1 = q|    \entry{snk1}{book}{}
       \strng{authorfullhash}{83330b0520b5d4ea57529a23b404d43d}
       \field{labelalpha}{vDoe}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
     \endentry
@@ -720,7 +720,7 @@ my $clone2 = q|    \entry{clone-snk1}{book}{}
       \strng{authorfullhash}{83330b0520b5d4ea57529a23b404d43d}
       \field{labelalpha}{vDoe}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{addendum}{add}
@@ -748,7 +748,7 @@ my $ent1 = q|    \entry{ent1}{book}{}
       \strng{authorfullhash}{b2536a425d549b46de5f21c4d468050a}
       \field{labelalpha}{SdB}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \true{singletitle}
       \field{labelnamesource}{author}
     \endentry
@@ -762,18 +762,18 @@ my $verb1 = q|    \\entry{verb1}{book}{}
            given={Arthur},
            giveni={A\\bibinitperiod}}}%
       }
-      \strng{namehash}{cac5a25f503e71f5ef28f474e14007b6}
-      \strng{fullhash}{cac5a25f503e71f5ef28f474e14007b6}
-      \strng{authornamehash}{cac5a25f503e71f5ef28f474e14007b6}
-      \strng{authorfullhash}{cac5a25f503e71f5ef28f474e14007b6}
-      \field{labelalpha}{All}
-      \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
-      \true{singletitle}
-      \field{labelnamesource}{author}
-      \verb{verba}
-      \verb \=y.\"a
-      \endverb
+      \\strng{namehash}{cac5a25f503e71f5ef28f474e14007b6}
+      \\strng{fullhash}{cac5a25f503e71f5ef28f474e14007b6}
+      \\strng{authornamehash}{cac5a25f503e71f5ef28f474e14007b6}
+      \\strng{authorfullhash}{cac5a25f503e71f5ef28f474e14007b6}
+      \\field{labelalpha}{All}
+      \\field{sortinit}{A}
+      \\field{sortinithash}{b685c7856330eaee22789815b49de9bb}
+      \\true{singletitle}
+      \\field{labelnamesource}{author}
+      \\verb{verba}
+      \\verb \=y.\"a
+      \\endverb
     \\endentry
 |;
 
diff --git a/t/biblatexml.t b/t/biblatexml.t
index b90353c9..57510dda 100755
--- a/t/biblatexml.t
+++ b/t/biblatexml.t
@@ -106,7 +106,7 @@ my $l1 = q|    \entry{bltx1}{misc}{useprefix=false}
       \strng{translatornamehash}{b44eba830fe9817fbe8e53c82f1cbe04}
       \strng{translatorfullhash}{b44eba830fe9817fbe8e53c82f1cbe04}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -151,7 +151,7 @@ my $l1 = q|    \entry{bltx1}{misc}{useprefix=false}
 
 my $l2 = q|    \entry{loopkey:a}{book}{}
       \field{sortinit}{0}
-      \field{sortinithash}{e07d8eeb31ded4126e0aa0cfcdcb5a8f}
+      \field{sortinithash}{990108227b3316c02842d895999a0165}
     \endentry
 |;
 
diff --git a/t/crossrefs.t b/t/crossrefs.t
index 7ac21a92..47f2940a 100755
--- a/t/crossrefs.t
+++ b/t/crossrefs.t
@@ -79,7 +79,7 @@ my $cr1 = q|    \entry{cr1}{inbook}{}
       \strng{editornamehash}{c129df5593fdaa7475548811bfbb227d}
       \strng{editorfullhash}{c129df5593fdaa7475548811bfbb227d}
       \field{sortinit}{G}
-      \field{sortinithash}{618d986594b7198ba52cf8b00d348f3f}
+      \field{sortinithash}{1c854ef9177a91bf894e66485bdbd3ed}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -125,7 +125,7 @@ my $cr2 = q|    \entry{cr2}{inbook}{}
       \strng{editornamehash}{c129df5593fdaa7475548811bfbb227d}
       \strng{editorfullhash}{c129df5593fdaa7475548811bfbb227d}
       \field{sortinit}{F}
-      \field{sortinithash}{276475738cc058478c1677046f857703}
+      \field{sortinithash}{c6a7d9913bbd7b20ea954441c0460b78}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -158,7 +158,7 @@ my $cr_m = q|    \entry{cr_m}{book}{}
       \strng{editornamehash}{c129df5593fdaa7475548811bfbb227d}
       \strng{editorfullhash}{c129df5593fdaa7475548811bfbb227d}
       \field{sortinit}{G}
-      \field{sortinithash}{618d986594b7198ba52cf8b00d348f3f}
+      \field{sortinithash}{1c854ef9177a91bf894e66485bdbd3ed}
       \true{crossrefsource}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
@@ -193,7 +193,7 @@ my $cr3 = q|    \entry{cr3}{inbook}{}
       \strng{editornamehash}{a1f5c22413396d599ec766725b226735}
       \strng{editorfullhash}{a1f5c22413396d599ec766725b226735}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -235,7 +235,7 @@ my $cr4 = q|    \entry{cr4}{inbook}{}
       \strng{editornamehash}{6ea89bd4958743a20b70fe17647d6af5}
       \strng{editorfullhash}{6ea89bd4958743a20b70fe17647d6af5}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -265,7 +265,7 @@ my $crt = q|    \entry{crt}{book}{}
       \strng{editornamehash}{a1f5c22413396d599ec766725b226735}
       \strng{editorfullhash}{a1f5c22413396d599ec766725b226735}
       \field{sortinit}{B}
-      \field{sortinithash}{5f6fa000f686ee5b41be67ba6ff7962d}
+      \field{sortinithash}{4ecbea03efd0532989d3836d1a048c32}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \field{title}{Beasts of the Burbling Burns}
@@ -297,7 +297,7 @@ my $cr6 = q|    \entry{cr6}{inproceedings}{}
       \strng{editornamehash}{344a7f427fb765610ef96eb7bce95257}
       \strng{editorfullhash}{344a7f427fb765610ef96eb7bce95257}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{uniquetitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -344,7 +344,7 @@ my $cr7 = q|    \entry{cr7}{inbook}{}
       \strng{bookauthornamehash}{91a1dd4aeed3c4ec29ca74c4e778be5f}
       \strng{bookauthorfullhash}{91a1dd4aeed3c4ec29ca74c4e778be5f}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{uniquetitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -376,7 +376,7 @@ my $cr8 = q|    \entry{cr8}{incollection}{}
       \strng{authornamehash}{3d449e56eb3ca1ae80dc99a18d689795}
       \strng{authorfullhash}{3d449e56eb3ca1ae80dc99a18d689795}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -404,7 +404,7 @@ my $xr1 = q|    \entry{xr1}{inbook}{}
       \strng{authornamehash}{e0ecc4fc668ee499d1afba44e1ac064d}
       \strng{authorfullhash}{e0ecc4fc668ee499d1afba44e1ac064d}
       \field{sortinit}{Z}
-      \field{sortinithash}{35589aa085e881766b72503e53fd4c97}
+      \field{sortinithash}{fdda4caaa6b5fa63e0c081dcb159543a}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -431,7 +431,7 @@ my $xr2 = q|    \entry{xr2}{inbook}{}
       \strng{authornamehash}{6afa09374ecfd6b394ce714d2d9709c7}
       \strng{authorfullhash}{6afa09374ecfd6b394ce714d2d9709c7}
       \field{sortinit}{I}
-      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
+      \field{sortinithash}{25e99d37ba90f7c4fb20baf4e310faf3}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -461,7 +461,7 @@ my $xrm = q|    \entry{xrm}{book}{}
       \strng{editornamehash}{809950f9b59ae207092b909a19dcb27b}
       \strng{editorfullhash}{809950f9b59ae207092b909a19dcb27b}
       \field{sortinit}{C}
-      \field{sortinithash}{095692fd22cc3c74d7fe223d02314dbd}
+      \field{sortinithash}{59f25d509f3381b07695554a9f35ecb2}
       \true{xrefsource}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
@@ -484,7 +484,7 @@ my $xr3 = q|    \entry{xr3}{inbook}{}
       \strng{authornamehash}{9788055665b9bb4b37c776c3f6b74f16}
       \strng{authorfullhash}{9788055665b9bb4b37c776c3f6b74f16}
       \field{sortinit}{N}
-      \field{sortinithash}{1163c28585427c673ad5a010cbf82f52}
+      \field{sortinithash}{925374ca63e7594de7fafdb83e64d41d}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -513,7 +513,7 @@ my $xrt = q|    \entry{xrt}{book}{}
       \strng{editornamehash}{bf7d6b02f3e073913e5bfe5059508dd5}
       \strng{editorfullhash}{bf7d6b02f3e073913e5bfe5059508dd5}
       \field{sortinit}{K}
-      \field{sortinithash}{4c244ceae61406cdc0cc2ce1cb1ff703}
+      \field{sortinithash}{a7d5b3aec5a0890aae7baf85a209abfc}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \field{title}{Kings, Cork and Calculation}
@@ -536,7 +536,7 @@ my $xr4 = q|    \entry{xr4}{inbook}{}
       \strng{authornamehash}{7804ffef086c0c4686c235807f5cb502}
       \strng{authorfullhash}{7804ffef086c0c4686c235807f5cb502}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{origyear}{1933}
@@ -561,7 +561,7 @@ my $mxr = q|    \entry{mxr}{inbook}{}
       \strng{authornamehash}{7804ffef086c0c4686c235807f5cb502}
       \strng{authorfullhash}{7804ffef086c0c4686c235807f5cb502}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{origyear}{1933}
@@ -583,7 +583,7 @@ my $mcr = q|    \entry{mcr}{inbook}{}
       \strng{authornamehash}{7804ffef086c0c4686c235807f5cb502}
       \strng{authorfullhash}{7804ffef086c0c4686c235807f5cb502}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{origyear}{1933}
@@ -614,7 +614,7 @@ my $ccr1 = q|    \entry{ccr2}{book}{}
       \strng{editornamehash}{cfee758a1c82df2e26af1985e061bb0a}
       \strng{editorfullhash}{cfee758a1c82df2e26af1985e061bb0a}
       \field{sortinit}{V}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \true{uniquetitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -646,7 +646,7 @@ my $ccr2 = q|    \entry{ccr3}{inbook}{}
       \strng{editornamehash}{cfee758a1c82df2e26af1985e061bb0a}
       \strng{editorfullhash}{cfee758a1c82df2e26af1985e061bb0a}
       \field{sortinit}{P}
-      \field{sortinithash}{24100cef455d7974167575052c29146e}
+      \field{sortinithash}{c0a4896d0e424f9ca4d7f14f2b3428e7}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \field{booktitle}{Misc etc.}
@@ -678,7 +678,7 @@ my $ccr3 = q|    \entry{ccr4}{inbook}{}
 
 my $s1 = q|    \entry{s1}{inbook}{}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \strng{crossref}{s2}
@@ -708,7 +708,7 @@ my $xc2 = q|    \entry{xc2}{inbook}{}
       \strng{bookauthornamehash}{1a0f7d518cccdad859a74412ef956474}
       \strng{bookauthorfullhash}{1a0f7d518cccdad859a74412ef956474}
       \field{sortinit}{C}
-      \field{sortinithash}{095692fd22cc3c74d7fe223d02314dbd}
+      \field{sortinithash}{59f25d509f3381b07695554a9f35ecb2}
       \true{xrefsource}
       \field{labelnamesource}{author}
       \field{booktitle}{Title}
@@ -717,7 +717,7 @@ my $xc2 = q|    \entry{xc2}{inbook}{}
 
 my $b1 = q|    \entry{b1}{inbook}{}
       \field{sortinit}{2}
-      \field{sortinithash}{0aa614ace9f3a40ef5a67e7f7a184048}
+      \field{sortinithash}{8343b463aacf48517c044b4d2c9c45ed}
       \strng{crossref}{b2}
       \field{day}{3}
       \field{month}{3}
@@ -748,7 +748,7 @@ my $sup1 = q|    \entry{sup1}{mvbook}{}
       \strng{authornamehash}{556c8dba145b472e6a8598d506f7cbe2}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \true{crossrefsource}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -770,7 +770,7 @@ my $sup2 = q|    \entry{sup2}{book}{}
       \strng{authornamehash}{556c8dba145b472e6a8598d506f7cbe2}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/dateformats.t b/t/dateformats.t
index d5437573..9bdf7f4c 100755
--- a/t/dateformats.t
+++ b/t/dateformats.t
@@ -84,7 +84,7 @@ my $l13c = q|    \entry{L13}{book}{}
       \strng{authornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -118,7 +118,7 @@ my $l14 = q|    \entry{L14}{book}{}
       \strng{authornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{3}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -156,7 +156,7 @@ my $l15 = q|    \entry{L15}{book}{}
       \strng{authornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{4}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -186,7 +186,7 @@ my $l16 = q|    \entry{L16}{proceedings}{}
       \strng{editornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{1}
       \field{labeldatesource}{event}
       \field{labelnamesource}{editor}
@@ -221,7 +221,7 @@ my $l17 = q|    \entry{L17}{proceedings}{}
       \strng{editornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{5}
       \field{labeldatesource}{}
       \field{labelnamesource}{editor}
@@ -276,7 +276,7 @@ my $l17c = q|    \entry{L17}{proceedings}{}
       \strng{editornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{orig}
       \field{labelnamesource}{editor}
       \field{labeltitlesource}{title}
@@ -330,7 +330,7 @@ my $l17e = q|    \entry{L17}{proceedings}{}
       \strng{editornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{event}
       \field{labelnamesource}{editor}
       \field{labeltitlesource}{title}
@@ -437,7 +437,7 @@ eq_or_diff($bibentries->entry('L17')->get_labeldate_info->{field}{source}, 'pubs
 
 my $era1 = q|    \entry{era1}{book}{}
       \field{sortinit}{3}
-      \field{sortinithash}{197da6d6c34c6b20ce45c4d4baace5a4}
+      \field{sortinithash}{7e21a31d9ef6ddc352701d4fa590bfbc}
       \field{labeldatesource}{}
       \field{month}{2}
       \field{origendyear}{219}
@@ -451,7 +451,7 @@ my $era1 = q|    \entry{era1}{book}{}
 
 my $era2 = q|    \entry{era2}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{eventyear}{249}
       \field{origendyear}{44}
@@ -466,7 +466,7 @@ my $era2 = q|    \entry{era2}{book}{}
 
 my $era3 = q|    \entry{era3}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{eventday}{2}
       \field{eventmonth}{3}
@@ -481,7 +481,7 @@ my $era3 = q|    \entry{era3}{book}{}
 
 my $era4 = q|    \entry{era4}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{eventyear}{1565}
       \field{origendyear}{1488}
@@ -504,7 +504,7 @@ my $era4 = q|    \entry{era4}{book}{}
 
 my $time1 = q|    \entry{time1}{book}{}
       \field{sortinit}{2}
-      \field{sortinithash}{0aa614ace9f3a40ef5a67e7f7a184048}
+      \field{sortinithash}{8343b463aacf48517c044b4d2c9c45ed}
       \field{labeldatesource}{}
       \field{day}{3}
       \field{eventday}{3}
@@ -534,7 +534,7 @@ my $time1 = q|    \entry{time1}{book}{}
 
 my $range1 = q|    \entry{range1}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{endyear}{}
       \field{eventendyear}{}
@@ -555,7 +555,7 @@ my $range1 = q|    \entry{range1}{book}{}
 
 my $range2 = q|    \entry{range2}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{endyear}{}
       \field{eventendyear}{1565}
@@ -576,7 +576,7 @@ my $range2 = q|    \entry{range2}{book}{}
 
 my $season1 = q|    \entry{season1}{book}{}
       \field{sortinit}{2}
-      \field{sortinithash}{0aa614ace9f3a40ef5a67e7f7a184048}
+      \field{sortinithash}{8343b463aacf48517c044b4d2c9c45ed}
       \field{labeldatesource}{}
       \field{eventseason}{autumn}
       \field{eventyear}{2002}
@@ -589,7 +589,7 @@ my $season1 = q|    \entry{season1}{book}{}
 
 my $unspec1 = q|    \entry{unspec1}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{endyear}{1999}
       \field{eventendyear}{1999}
@@ -623,7 +623,7 @@ my $unspec1 = q|    \entry{unspec1}{book}{}
 
 my $unspec2 = q|    \entry{unspec2}{book}{}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{day}{1}
       \field{endday}{31}
diff --git a/t/encoding.t b/t/encoding.t
index e0d635be..f9c9b09d 100755
--- a/t/encoding.t
+++ b/t/encoding.t
@@ -49,7 +49,7 @@ my $encode1 = q|    \entry{testŠ}{book}{}
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{year}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -76,7 +76,7 @@ my $encode2 = q|    \entry{test1}{book}{}
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{year}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -103,7 +103,7 @@ my $encode3 = q|    \entry{test1}{book}{}
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{year}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -130,7 +130,7 @@ my $encode5 = q|    \entry{test}{book}{}
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{year}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -157,7 +157,7 @@ my $encode6 = q|    \entry{test}{book}{}
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{year}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -184,7 +184,7 @@ my $encode7 = q|    \entry{test}{book}{}
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{year}
       \true{singletitle}
       \field{labelnamesource}{author}
diff --git a/t/full-bbl.t b/t/full-bbl.t
index 9bf51215..3de67708 100755
--- a/t/full-bbl.t
+++ b/t/full-bbl.t
@@ -29,8 +29,8 @@ my $stdout;
 my $stderr;
 
 run3  [ $perl, 'bin/biber', '--noconf', '--nolog', "--output-file=$bbl", 't/tdata/full-bbl.bcf' ], \undef, \$stdout, \$stderr;
-# say $stdout;
-# say $stderr;
+#say $stdout;
+#say $stderr;
 
 is($? >> 8, 0, 'Full test has zero exit status');
 ok(compare($bbl, 't/tdata/full-bbl.bbl') == 0, 'Testing lossort case and sortinit for macros');
diff --git a/t/full-bblxml.t b/t/full-bblxml.t
index 528f224d..ee317d4d 100755
--- a/t/full-bblxml.t
+++ b/t/full-bblxml.t
@@ -29,8 +29,8 @@ my $stdout;
 my $stderr;
 
 run3  [ $perl, 'bin/biber', '--noconf', '--nolog', '--output-format=bblxml', "--output-file=$bblxml", '--validate-bblxml', 't/tdata/full-bblxml.bcf' ], \undef, \$stdout, \$stderr;
-# say $stdout;
-# say $stderr;
+#say $stdout;
+#say $stderr;
 
 is($? >> 8, 0, 'Full test has zero exit status');
 
diff --git a/t/names.t b/t/names.t
index e6f8d955..713e6923 100755
--- a/t/names.t
+++ b/t/names.t
@@ -265,7 +265,7 @@ my $l1 = q|    \entry{L1}{book}{}
       \strng{authornamehash}{72287a68c1714cb1b9f4ab9e03a88b96}
       \strng{authorfullhash}{72287a68c1714cb1b9f4ab9e03a88b96}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -283,7 +283,7 @@ my $l2 = q|    \entry{L2}{book}{}
       \strng{authornamehash}{1c867a2b5ceb243bab70afb18702dc04}
       \strng{authorfullhash}{1c867a2b5ceb243bab70afb18702dc04}
       \field{sortinit}{B}
-      \field{sortinithash}{5f6fa000f686ee5b41be67ba6ff7962d}
+      \field{sortinithash}{4ecbea03efd0532989d3836d1a048c32}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -301,7 +301,7 @@ my $l3 = q|    \entry{L3}{book}{}
       \strng{authornamehash}{cecd18116c43ee86e5a136b6e0362948}
       \strng{authorfullhash}{cecd18116c43ee86e5a136b6e0362948}
       \field{sortinit}{C}
-      \field{sortinithash}{095692fd22cc3c74d7fe223d02314dbd}
+      \field{sortinithash}{59f25d509f3381b07695554a9f35ecb2}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -319,7 +319,7 @@ my $l4 = q|    \entry{L4}{book}{}
       \strng{authornamehash}{675883f3aca7c6069c0b154d47af4c86}
       \strng{authorfullhash}{675883f3aca7c6069c0b154d47af4c86}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -339,7 +339,7 @@ my $l5 = q|    \entry{L5}{book}{}
       \strng{authornamehash}{c6b9d281cc1ff3f35570f76f463d4244}
       \strng{authorfullhash}{c6b9d281cc1ff3f35570f76f463d4244}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -359,7 +359,7 @@ my $l6 = q|    \entry{L6}{book}{}
       \strng{authornamehash}{dd96e3fc645eb4685988366f233403df}
       \strng{authorfullhash}{dd96e3fc645eb4685988366f233403df}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -379,7 +379,7 @@ my $l7 = q|    \entry{L7}{book}{}
       \strng{authornamehash}{1e802cc32f10930a9567712b8febdf19}
       \strng{authorfullhash}{1e802cc32f10930a9567712b8febdf19}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -399,7 +399,7 @@ my $l8 = q|    \entry{L8}{book}{}
       \strng{authornamehash}{076a9b62b331eb2cdfba234d9ad7bca9}
       \strng{authorfullhash}{076a9b62b331eb2cdfba234d9ad7bca9}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -417,7 +417,7 @@ my $l9 = q|    \entry{L9}{book}{}
       \strng{authornamehash}{1734924c4c55de5bb18d020c34a5249e}
       \strng{authorfullhash}{1734924c4c55de5bb18d020c34a5249e}
       \field{sortinit}{I}
-      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
+      \field{sortinithash}{25e99d37ba90f7c4fb20baf4e310faf3}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -438,7 +438,7 @@ my $l10 = q|    \entry{L10}{book}{}
       \strng{authornamehash}{37b4325752e394ddfb2fc810f6c88e27}
       \strng{authorfullhash}{37b4325752e394ddfb2fc810f6c88e27}
       \field{sortinit}{J}
-      \field{sortinithash}{c86bd6cced82a15683b396c2169909ef}
+      \field{sortinithash}{ec3950a647c092421b9fcca6d819504a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -459,7 +459,7 @@ my $l10a = q|    \entry{L10a}{book}{}
       \strng{authornamehash}{264cb53d2295644c1c99523e254d9b0e}
       \strng{authorfullhash}{264cb53d2295644c1c99523e254d9b0e}
       \field{sortinit}{P}
-      \field{sortinithash}{24100cef455d7974167575052c29146e}
+      \field{sortinithash}{c0a4896d0e424f9ca4d7f14f2b3428e7}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -482,7 +482,7 @@ my $l11 = q|    \entry{L11}{book}{}
       \strng{authornamehash}{c536dd808dc9193fda59ba1ff2afb38f}
       \strng{authorfullhash}{c536dd808dc9193fda59ba1ff2afb38f}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -502,7 +502,7 @@ my $l12 = q|    \entry{L12}{book}{}
       \strng{authornamehash}{7e7640f3cb87a1cf11a86307a186ec0f}
       \strng{authorfullhash}{7e7640f3cb87a1cf11a86307a186ec0f}
       \field{sortinit}{d}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \true{uniqueprimaryauthor}
       \field{labelnamesource}{author}
     \endentry
@@ -521,7 +521,7 @@ my $l13 = q|    \entry{L13}{book}{}
       \strng{authornamehash}{5e79da6869afaf0d38e01285b494d555}
       \strng{authorfullhash}{5e79da6869afaf0d38e01285b494d555}
       \field{sortinit}{V}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -539,7 +539,7 @@ my $l14 = q|    \entry{L14}{book}{}
       \strng{authornamehash}{2319907d9a5d5dd46da77879bdb7e609}
       \strng{authorfullhash}{2319907d9a5d5dd46da77879bdb7e609}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -559,7 +559,7 @@ my $l15 = q|    \entry{L15}{book}{}
       \strng{authornamehash}{b30b0fc69681fd11ad5d75a880124091}
       \strng{authorfullhash}{b30b0fc69681fd11ad5d75a880124091}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -579,7 +579,7 @@ my $l16 = q|    \entry{L16}{book}{}
       \strng{authornamehash}{2b17c50dc666b9cc73d132da9ef08c7b}
       \strng{authorfullhash}{2b17c50dc666b9cc73d132da9ef08c7b}
       \field{sortinit}{v}
-      \field{sortinithash}{555737dafdcf1396ebfeae5822e5bde2}
+      \field{sortinithash}{d18f5ce25ce0b5ca7f924e3f6c04870e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -597,7 +597,7 @@ my $l17 = q|    \entry{L17}{book}{}
       \strng{authornamehash}{766d5329cf995fcc7c1cef19de2a2ae8}
       \strng{authorfullhash}{766d5329cf995fcc7c1cef19de2a2ae8}
       \field{sortinit}{L}
-      \field{sortinithash}{7bba64db83423e3c29ad597f3b682cf3}
+      \field{sortinithash}{872351f18d0f736066eda0bf18bfa4f7}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -615,7 +615,7 @@ my $l18 = q|    \entry{L18}{book}{}
       \strng{authornamehash}{58620d2c7d6839bac23306c732c563fb}
       \strng{authorfullhash}{58620d2c7d6839bac23306c732c563fb}
       \field{sortinit}{L}
-      \field{sortinithash}{7bba64db83423e3c29ad597f3b682cf3}
+      \field{sortinithash}{872351f18d0f736066eda0bf18bfa4f7}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -633,7 +633,7 @@ my $l19 = q|    \entry{L19}{book}{}
       \strng{authornamehash}{83caa52f21f97e572dd3267bdf62978a}
       \strng{authorfullhash}{83caa52f21f97e572dd3267bdf62978a}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -651,7 +651,7 @@ my $l19a = q|    \entry{L19a}{book}{}
       \strng{authornamehash}{0963f6904ccfeaac2770c5882a587001}
       \strng{authorfullhash}{0963f6904ccfeaac2770c5882a587001}
       \field{sortinit}{L}
-      \field{sortinithash}{7bba64db83423e3c29ad597f3b682cf3}
+      \field{sortinithash}{872351f18d0f736066eda0bf18bfa4f7}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -670,7 +670,7 @@ my $l20 = q|    \entry{L20}{book}{}
       \strng{authornamehash}{fdaa0936724be89ef8bd16cf02e08c74}
       \strng{authorfullhash}{fdaa0936724be89ef8bd16cf02e08c74}
       \field{sortinit}{F}
-      \field{sortinithash}{276475738cc058478c1677046f857703}
+      \field{sortinithash}{c6a7d9913bbd7b20ea954441c0460b78}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -688,7 +688,7 @@ my $l21 = q|    \entry{L21}{book}{}
       \strng{authornamehash}{4389a3c0dc7da74487b50808ba9436ad}
       \strng{authorfullhash}{4389a3c0dc7da74487b50808ba9436ad}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -706,7 +706,7 @@ my $l22u = q|    \entry{L22}{book}{}
       \strng{authornamehash}{e58b861545799d0eaf883402a882126e}
       \strng{authorfullhash}{e58b861545799d0eaf883402a882126e}
       \field{sortinit}{Š}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -725,7 +725,7 @@ my $l22 = q|    \entry{L22}{book}{}
       \strng{authornamehash}{e58b861545799d0eaf883402a882126e}
       \strng{authorfullhash}{e58b861545799d0eaf883402a882126e}
       \field{sortinit}{\v{S}}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -744,7 +744,7 @@ my $l23 = q|    \entry{L23}{book}{}
       \strng{authornamehash}{4389a3c0dc7da74487b50808ba9436ad}
       \strng{authorfullhash}{4389a3c0dc7da74487b50808ba9436ad}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -762,7 +762,7 @@ my $l24 = q|    \entry{L24}{book}{}
       \strng{authornamehash}{e58b861545799d0eaf883402a882126e}
       \strng{authorfullhash}{e58b861545799d0eaf883402a882126e}
       \field{sortinit}{Š}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -778,7 +778,7 @@ my $l25 = q|    \entry{L25}{book}{}
       \strng{authornamehash}{7069367d4a4f37ffb0377e3830e98ed0}
       \strng{authorfullhash}{7069367d4a4f37ffb0377e3830e98ed0}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -794,7 +794,7 @@ my $l26 = q|    \entry{L26}{book}{}
       \strng{authornamehash}{d176a8af5ce1c45cb06875c4433f2fe2}
       \strng{authorfullhash}{d176a8af5ce1c45cb06875c4433f2fe2}
       \field{sortinit}{S}
-      \field{sortinithash}{3c1547c63380458f8ca90e40ed14b83e}
+      \field{sortinithash}{fd1e7c5ab79596b13dbbb67f8d70fb5a}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -802,7 +802,7 @@ my $l26 = q|    \entry{L26}{book}{}
 # Malformed anyway but a decent test
 my $l28 = q|    \entry{L28}{book}{}
       \field{sortinit}{0}
-      \field{sortinithash}{e07d8eeb31ded4126e0aa0cfcdcb5a8f}
+      \field{sortinithash}{990108227b3316c02842d895999a0165}
       \warn{\item Name "Deux et al.,, O." is malformed (consecutive commas): skipping name}
     \endentry
 |;
@@ -819,7 +819,7 @@ my $l29 = q|    \entry{L29}{book}{}
       \strng{authornamehash}{59a5e43a502767d00e589eb29f863728}
       \strng{authorfullhash}{59a5e43a502767d00e589eb29f863728}
       \field{sortinit}{U}
-      \field{sortinithash}{1554584e7f69d2b4fe0a645e73d50194}
+      \field{sortinithash}{8145509bd2718876fc77d31fd2cde117}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -855,7 +855,7 @@ my $l31 = q|    \entry{L31}{book}{}
       \strng{translatornamehash}{29c3ff92fff79d09a8b44d2f775de0b1}
       \strng{translatorfullhash}{29c3ff92fff79d09a8b44d2f775de0b1}
       \field{sortinit}{\~{Z}}
-      \field{sortinithash}{35589aa085e881766b72503e53fd4c97}
+      \field{sortinithash}{fdda4caaa6b5fa63e0c081dcb159543a}
       \true{uniqueprimaryauthor}
       \field{labelnamesource}{author}
     \endentry
diff --git a/t/options.t b/t/options.t
index 1a962429..a40e3975 100755
--- a/t/options.t
+++ b/t/options.t
@@ -97,7 +97,7 @@ my $l1 = q|    \entry{L1}{book}{}
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -130,7 +130,7 @@ my $l2 = q|    \entry{L2}{book}{maxcitenames=3,maxbibnames=3,maxitems=2}
       \strng{authornamehash}{19eec87c959944d6d9c72434a42856ba}
       \strng{authorfullhash}{19eec87c959944d6d9c72434a42856ba}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -158,7 +158,7 @@ my $l3 = q|    \entry{L3}{book}{blah=10}
       \strng{authornamehash}{490250da1f3b92580d97563dc96c6c84}
       \strng{authorfullhash}{490250da1f3b92580d97563dc96c6c84}
       \field{sortinit}{B}
-      \field{sortinithash}{5f6fa000f686ee5b41be67ba6ff7962d}
+      \field{sortinithash}{4ecbea03efd0532989d3836d1a048c32}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/related-entries.t b/t/related-entries.t
index f107f8ec..2b31d3f6 100755
--- a/t/related-entries.t
+++ b/t/related-entries.t
@@ -55,7 +55,7 @@ my $k1 = q|    \entry{key1}{article}{}
       \strng{authornamehash}{a517747c3d12f99244ae598910d979c5}
       \strng{authorfullhash}{a517747c3d12f99244ae598910d979c5}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -90,7 +90,7 @@ my $k2 = q|    \entry{key2}{inbook}{}
       \strng{authornamehash}{a517747c3d12f99244ae598910d979c5}
       \strng{authorfullhash}{a517747c3d12f99244ae598910d979c5}
       \field{sortinit}{2}
-      \field{sortinithash}{0aa614ace9f3a40ef5a67e7f7a184048}
+      \field{sortinithash}{8343b463aacf48517c044b4d2c9c45ed}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -229,7 +229,7 @@ my $kck4 = q|    \entry{caf8e34be07426ae7127c1b4829983c1}{inbook}{dataonly,useed
 
 my $c1 = q|    \entry{c1}{book}{}
       \field{sortinit}{3}
-      \field{sortinithash}{197da6d6c34c6b20ce45c4d4baace5a4}
+      \field{sortinithash}{7e21a31d9ef6ddc352701d4fa590bfbc}
       \field{related}{9ab62b5ef34a985438bfdf7ee0102229}
     \endentry
 |;
diff --git a/t/remote-files.t b/t/remote-files.t
index b5acd93e..8d9ae248 100755
--- a/t/remote-files.t
+++ b/t/remote-files.t
@@ -80,7 +80,7 @@ my $cu1 = q|    \entry{citeulike:8283461}{article}{}
       \strng{authornamehash}{7f19319e09aa3239f02eb31ec7a4aa8b}
       \strng{authorfullhash}{ee363ce5e21ebe022f83aae896dd47f9}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -119,7 +119,7 @@ my $dl1 = q|    \entry{AbdelbarH98}{article}{}
       \strng{authornamehash}{01599a4cb58316d64208b12a07741765}
       \strng{authorfullhash}{01599a4cb58316d64208b12a07741765}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -163,7 +163,7 @@ my $ssl = q|    \entry{merleau-ponty_philosophe_2010}{incollection}{}
       \strng{editornamehash}{ff5f90046157eecef0c22da4dac6486e}
       \strng{editorfullhash}{ff5f90046157eecef0c22da4dac6486e}
       \field{sortinit}{M}
-      \field{sortinithash}{c26a05ef03e4429073ed5c825140fac3}
+      \field{sortinithash}{2684bec41e9697b92699b46491061da2}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/set-dynamic.t b/t/set-dynamic.t
index 858d1e50..b6fc5694 100755
--- a/t/set-dynamic.t
+++ b/t/set-dynamic.t
@@ -60,7 +60,7 @@ my $string1 = q|    \entry{DynSet}{set}{}
       \strng{authornamehash}{252caa7921a061ca92087a1a52f15b78}
       \strng{authorfullhash}{252caa7921a061ca92087a1a52f15b78}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -85,7 +85,7 @@ my $string2 = q|    \entry{Dynamic1}{book}{}
       \strng{authornamehash}{252caa7921a061ca92087a1a52f15b78}
       \strng{authorfullhash}{252caa7921a061ca92087a1a52f15b78}
       \field{sortinit}{8}
-      \field{sortinithash}{2f1b0e69e7617f99664f1cd00bb9603c}
+      \field{sortinithash}{ff392057b609ad7b4de0b8e2cc30b687}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -110,7 +110,7 @@ my $string3 = q|    \entry{Dynamic2}{book}{}
       \strng{authornamehash}{894a5fe6de820f5dcce84a65581667f4}
       \strng{authorfullhash}{894a5fe6de820f5dcce84a65581667f4}
       \field{sortinit}{9}
-      \field{sortinithash}{2999ebab86052e6d71b434385f8b4ed2}
+      \field{sortinithash}{5a7694b2c36c7534b22948c31e4b7920}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -134,7 +134,7 @@ my $string4 = q|    \entry{Dynamic3}{book}{}
       \strng{authornamehash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \strng{authorfullhash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -159,7 +159,7 @@ my $string5 = q|    \entry{Dynamic3}{book}{}
       \strng{authornamehash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \strng{authorfullhash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/set-legacy.t b/t/set-legacy.t
index 1f9bfd19..dd072b09 100755
--- a/t/set-legacy.t
+++ b/t/set-legacy.t
@@ -56,7 +56,7 @@ my $string1 = q|    \entry{Elias1955}{set}{}
       \strng{authornamehash}{bdd4981ffb5a62685c993d6f9dec4c23}
       \strng{authorfullhash}{bdd4981ffb5a62685c993d6f9dec4c23}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/set-static.t b/t/set-static.t
index 98f3e193..34df8692 100755
--- a/t/set-static.t
+++ b/t/set-static.t
@@ -56,7 +56,7 @@ my $string1 = q|    \entry{Static1}{set}{}
       \strng{authornamehash}{43874d80d7ce68027102819f16c47df1}
       \strng{authorfullhash}{43874d80d7ce68027102819f16c47df1}
       \field{sortinit}{2}
-      \field{sortinithash}{0aa614ace9f3a40ef5a67e7f7a184048}
+      \field{sortinithash}{8343b463aacf48517c044b4d2c9c45ed}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -80,7 +80,7 @@ my $string2 = q|    \entry{Static2}{book}{}
       \strng{authornamehash}{43874d80d7ce68027102819f16c47df1}
       \strng{authorfullhash}{43874d80d7ce68027102819f16c47df1}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -147,7 +147,7 @@ my $string5 = q|    \entry{Static2}{book}{}
       \strng{authornamehash}{43874d80d7ce68027102819f16c47df1}
       \strng{authorfullhash}{43874d80d7ce68027102819f16c47df1}
       \field{sortinit}{1}
-      \field{sortinithash}{27a2bc5dfb9ed0a0422134d636544b5d}
+      \field{sortinithash}{53e85f4cf497d2a4e1d51786552bcbf1}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/skips.t b/t/skips.t
index 66a4f6e7..3ea41a14 100755
--- a/t/skips.t
+++ b/t/skips.t
@@ -59,7 +59,7 @@ my $set1 = q|    \entry{seta}{set}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{1}
       \field{labeldatesource}{}
       \field{extraalpha}{1}
@@ -85,7 +85,7 @@ my $set2 = q|    \entry{set:membera}{book}{}
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -110,7 +110,7 @@ my $set3 = q|    \entry{set:memberb}{book}{}
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -134,7 +134,7 @@ my $set4 = q|    \entry{set:memberc}{book}{}
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -158,7 +158,7 @@ my $noset1 = q|    \entry{noseta}{book}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{2}
       \field{labeldatesource}{}
       \field{extraalpha}{2}
@@ -184,7 +184,7 @@ my $noset2 = q|    \entry{nosetb}{book}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{3}
       \field{labeldatesource}{}
       \field{extraalpha}{3}
@@ -210,7 +210,7 @@ my $noset3 = q|    \entry{nosetc}{book}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extrayear}{4}
       \field{labeldatesource}{}
       \field{extraalpha}{4}
@@ -241,7 +241,7 @@ my $sk4 = q|    \entry{skip4}{article}{dataonly}
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/sort-complex.t b/t/sort-complex.t
index 77f515c9..30115b79 100755
--- a/t/sort-complex.t
+++ b/t/sort-complex.t
@@ -119,7 +119,7 @@ my $l4 = q|    \entry{L4}{book}{}
       \strng{authorfullhash}{6eb389989020e8246fee90ac93fcecbe}
       \field{labelalpha}{Doe\textbf{+}95}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -148,7 +148,7 @@ my $l1 = q|    \entry{L1}{book}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe95}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -177,7 +177,7 @@ my $l2 = q|    \entry{L2}{book}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe95}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extraalpha}{3}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -206,7 +206,7 @@ my $l3 = q|    \entry{L3}{book}{}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{Doe95}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -237,7 +237,7 @@ my $l5 = q|    \entry{L5}{book}{}
       \strng{authorfullhash}{6eb389989020e8246fee90ac93fcecbe}
       \field{labelalpha}{Doe\textbf{+}95}
       \field{sortinit}{D}
-      \field{sortinithash}{d10b5413de1f3d197b20897dd0d565bb}
+      \field{sortinithash}{78f7c4753a2004675f316a80bdb31742}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/sortlists.t b/t/sortlists.t
index b9702e99..91a3a1f4 100755
--- a/t/sortlists.t
+++ b/t/sortlists.t
@@ -84,7 +84,7 @@ my $K11 = q|    \entry{K11}{book}{}
       \strng{authornamehash}{4edc280a0ef229f9c061e3b121b17482}
       \strng{authorfullhash}{4edc280a0ef229f9c061e3b121b17482}
       \field{sortinit}{a}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -113,7 +113,7 @@ my $K12 = q|    \entry{K12}{book}{}
       \strng{authornamehash}{a846a485fc9cbb59b0ebeedd6ac637e4}
       \strng{authorfullhash}{a846a485fc9cbb59b0ebeedd6ac637e4}
       \field{sortinit}{Z}
-      \field{sortinithash}{35589aa085e881766b72503e53fd4c97}
+      \field{sortinithash}{fdda4caaa6b5fa63e0c081dcb159543a}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/tdata/full-bbl.bbl b/t/tdata/full-bbl.bbl
index cedc3be8..0337f2c5 100644
--- a/t/tdata/full-bbl.bbl
+++ b/t/tdata/full-bbl.bbl
@@ -33,7 +33,7 @@
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{\emph{A}}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -57,7 +57,7 @@
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{\emph{A}}
       \field{sortinit}{A}
-      \field{sortinithash}{3248043b5fe8d0a34dab5ab6b8d4309b}
+      \field{sortinithash}{b685c7856330eaee22789815b49de9bb}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git a/t/tdata/full-bblxml.bblxml b/t/tdata/full-bblxml.bblxml
index 0acc09a8..9ad33027 100644
--- a/t/tdata/full-bblxml.bblxml
+++ b/t/tdata/full-bblxml.bblxml
@@ -45,7 +45,7 @@
         <bbl:field name="authorfullhash">562d8cda005e7c37bc696e0d0edc306e</bbl:field>
         <bbl:field name="labelalpha">Aks+06</bbl:field>
         <bbl:field name="sortinit">A</bbl:field>
-        <bbl:field name="sortinithash">3248043b5fe8d0a34dab5ab6b8d4309b</bbl:field>
+        <bbl:field name="sortinithash">b685c7856330eaee22789815b49de9bb</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="indextitle">Effect of immobilization on catalytic characteristics</bbl:field>
@@ -75,7 +75,7 @@
         <bbl:field name="authorfullhash">9717d2d959a4b18727868ca7928c34b4</bbl:field>
         <bbl:field name="labelalpha">Ang02</bbl:field>
         <bbl:field name="sortinit">A</bbl:field>
-        <bbl:field name="sortinithash">3248043b5fe8d0a34dab5ab6b8d4309b</bbl:field>
+        <bbl:field name="sortinithash">b685c7856330eaee22789815b49de9bb</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
         <bbl:field name="annotation">A German article in a French journal. Apart from that, a typical article entry. Note the indextitle field</bbl:field>
@@ -115,7 +115,7 @@
         <bbl:field name="authorfullhash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="labelalpha">Gla61</bbl:field>
         <bbl:field name="sortinit">G</bbl:field>
-        <bbl:field name="sortinithash">618d986594b7198ba52cf8b00d348f3f</bbl:field>
+        <bbl:field name="sortinithash">1c854ef9177a91bf894e66485bdbd3ed</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">A set entry with three members discussing the standard model of particle physics.</bbl:field>
@@ -145,7 +145,7 @@
         <bbl:field name="authornamehash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="authorfullhash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="sortinit">G</bbl:field>
-        <bbl:field name="sortinithash">618d986594b7198ba52cf8b00d348f3f</bbl:field>
+        <bbl:field name="sortinithash">1c854ef9177a91bf894e66485bdbd3ed</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="journaltitle">Nucl. Phys.</bbl:field>
@@ -178,7 +178,7 @@
         <bbl:field name="authorfullhash">fcbea740aeb72b8e941e0d4aa9f6a9c6</bbl:field>
         <bbl:field name="labelalpha">Knu86</bbl:field>
         <bbl:field name="sortinit">K</bbl:field>
-        <bbl:field name="sortinithash">4c244ceae61406cdc0cc2ce1cb1ff703</bbl:field>
+        <bbl:field name="sortinithash">a7d5b3aec5a0890aae7baf85a209abfc</bbl:field>
         <bbl:field name="extraalpha">1</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
@@ -211,7 +211,7 @@
         <bbl:field name="authorfullhash">fcbea740aeb72b8e941e0d4aa9f6a9c6</bbl:field>
         <bbl:field name="labelalpha">Knu86</bbl:field>
         <bbl:field name="sortinit">K</bbl:field>
-        <bbl:field name="sortinithash">4c244ceae61406cdc0cc2ce1cb1ff703</bbl:field>
+        <bbl:field name="sortinithash">a7d5b3aec5a0890aae7baf85a209abfc</bbl:field>
         <bbl:field name="extraalpha">2</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
@@ -238,7 +238,7 @@
         <bbl:field name="authorfullhash">7c73b571a33320e72d2d87ff821579fd</bbl:field>
         <bbl:field name="labelalpha">Ree58</bbl:field>
         <bbl:field name="sortinit">R</bbl:field>
-        <bbl:field name="sortinithash">c15bc8eb6936bc6b3c8baa9e8575af53</bbl:field>
+        <bbl:field name="sortinithash">c7387613477035a752d935acfc3e3ea2</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">An article entry with a series and a volume field. Note the format of the series. If the value of the series field is an integer, this number is printed as an ordinal and the string *series is appended automatically</bbl:field>
@@ -285,7 +285,7 @@
         <bbl:field name="editornamehash">d0ec890e59163c24c111a08d2a4be982</bbl:field>
         <bbl:field name="editorfullhash">d0ec890e59163c24c111a08d2a4be982</bbl:field>
         <bbl:field name="sortinit">S</bbl:field>
-        <bbl:field name="sortinithash">3c1547c63380458f8ca90e40ed14b83e</bbl:field>
+        <bbl:field name="sortinithash">fd1e7c5ab79596b13dbbb67f8d70fb5a</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="booksubtitle">Relativistic groups and analyticity</bbl:field>
@@ -314,7 +314,7 @@
         <bbl:field name="authorfullhash">93f17724cbd884231ef39a73755ef5bb</bbl:field>
         <bbl:field name="labelalpha">Sho91</bbl:field>
         <bbl:field name="sortinit">S</bbl:field>
-        <bbl:field name="sortinithash">3c1547c63380458f8ca90e40ed14b83e</bbl:field>
+        <bbl:field name="sortinithash">fd1e7c5ab79596b13dbbb67f8d70fb5a</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">An article entry with series, volume, and number fields. Note the format of the series which is a localization key</bbl:field>
@@ -348,7 +348,7 @@
         <bbl:field name="authornamehash">15e144c72b0b9d10db00ddc6d37a19e6</bbl:field>
         <bbl:field name="authorfullhash">15e144c72b0b9d10db00ddc6d37a19e6</bbl:field>
         <bbl:field name="sortinit">W</bbl:field>
-        <bbl:field name="sortinithash">6d25b3eefe5aa2147d1f339686808918</bbl:field>
+        <bbl:field name="sortinithash">99e3ba1b3f78bb6f073e7fa7ac11636b</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="journaltitle">Phys. Rev. Lett.</bbl:field>
diff --git a/t/xdata.t b/t/xdata.t
index 5f7eba45..9664ed06 100644
--- a/t/xdata.t
+++ b/t/xdata.t
@@ -72,7 +72,7 @@ my $xd1 = q|    \entry{xd1}{book}{}
       \strng{authornamehash}{51db4bfd331cba22959ce2d224c517cd}
       \strng{authorfullhash}{51db4bfd331cba22959ce2d224c517cd}
       \field{sortinit}{E}
-      \field{sortinithash}{07bbd5a529b5beaa311df5be05b874bc}
+      \field{sortinithash}{fefc5210ef4721525b2a478df41efcd4}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{note}{A Note}
@@ -101,7 +101,7 @@ my $xd2 = q|    \entry{xd2}{book}{}
       \strng{authornamehash}{68539e0ce4922cc4957c6cabf35e6fc8}
       \strng{authorfullhash}{68539e0ce4922cc4957c6cabf35e6fc8}
       \field{sortinit}{P}
-      \field{sortinithash}{24100cef455d7974167575052c29146e}
+      \field{sortinithash}{c0a4896d0e424f9ca4d7f14f2b3428e7}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{abstract}{An abstract}
